client/package.json

```diff
@@ -46,21 +47,22 @@
   },
   "devDependencies": {
     "@testing-library/user-event": "^12.1.10",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
     "@types/react-router-dom": "^5.1.9",
     "@types/superagent": "^4.1.12",
     "@typescript-eslint/eslint-plugin": "^4.29.0",
     "@typescript-eslint/parser": "^4.29.0",
     "@vitejs/plugin-react": "^1.0.0",
-    "@types/node": "^12.0.0",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
     "cypress": "^8.2.0",
-    "typescript": "^4.1.2",
```

Comment:
This adds `react-cx` and `sass` as dependencies, and moves things around for some reason (?).